test/cbmc/proofs/MQTTAgent_Terminate/README.md

```diff
@@ -0,0 +1,20 @@
+MQTTAgent_Terminate proof
+==============
+
+This directory contains a memory safety proof for MQTTAgent_Terminate.
```

Comment:
Can we say anything about the coverage?  Approximate time to run?  What is tested vs. what is stubbed?

Reply:
Sure. That will be a good addition. Thanks Dan

Reply:
I have added a statement about the stubs used. I think the time to run will be depending on many factors such as system on which it is running and will be viewable from the results in the CI run. Coverage also will be changing depending on if there are any changes made to the API later. It will also be seen from the CI run. So those are not added in README, but added a mention about the stubs. Please let me know if you agree with this change.

Reply:
As discussed offline, added comments for these.